src/Controller/AbsenceBlockController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Controller;
+
+use App\Controller\BaseController;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Routing\Annotation\Route;
+
+use App\Model\AbsenceBlock;
+
+class AbsenceBlockController extends BaseController
+{
+    /**
+     * @Route("/absence/block", name="absence.block.index", methods={"GET"})
+     */
+    public function index(Request $request, Session $session)
+    {
+        $today = date('Y-m-d');
+
+        $queryBuilder = $this->entityManager->createQueryBuilder();
+
+        $query = $queryBuilder->select(array('a'))
+            ->from(AbsenceBlock::class, 'a')
+            ->orderBy('a.start', 'ASC', 'a.end', 'ASC')
+            ->getQuery();
+
+        $this->templateParams( array('block' => $query->getResult()) );
+
+        return $this->output('absenceBlock/index.html.twig');
+    }
+
+    /**
+     * @Route("/absence/block/add", name="absence.block.add", methods={"GET"})
+     */
+    public function add(Request $request)
+    {
+        $this->templateParams(array(
+            'id'    => null,
+            'start' => null,
+            'end'   => null,
+            'text'  => null,
+        ));
+
+        return $this->output('absenceBlock/edit.html.twig');
+    }
+
+    /**
+     * @Route("/absence/block/{id}", name="absence.block.edit", methods={"GET"})
+     */
+    public function edit(Request $request)
+    {
+        $id = $request->get('id');
+
+        $block = $this->entityManager->getRepository(AbsenceBlock::class)->findOneById($id);
```

Comment:
On peut simplifier par 
`$block = $this->entityManager->getRepository(AbsenceBlock::class)->find($id);`

Reply:
fixed by a913a36e4144640a9a040cb71f9acd9994bc1ccf